errors/public.go

```diff
@@ -60,3 +62,14 @@ func GetName(err error) string {
 		return "ErrUnknown"
 	}
 }
+
+// ElementError adds generic "element" information (file path, workflow job, step ID,
+// etc) to an error.
+type ElementError struct {
```

Comment:
Since this is one of the more public parts of the code. Can you clarify what an Element is?  The examples are good, but what is an element? Some discrete portion of analysis data for the various scorecard checks?